enclave/src/data/data.cc

```diff
@@ -281,6 +280,171 @@ DMatrix* DMatrix::Load(const std::string& uri,
   return dmat;
 }
 
+DMatrix* DMatrix::LoadMultiple(std::vector<const std::string>& uris,
+                       int num_uris,
```

Comment:
Do we need to pass `num_uris` in? Can be obtained directly from the vector, no?

Reply:
True. Fixed